### DISCUSSION TITLE
Livarno/Lidl RGB led light TS0505B not recognized

### DISCUSSION THREAD

**Discussion by Like-electronics** [Koenkk/zigbee2mqtt#11553] [Devices]
I bought this lamp last friday but i cannot get it recognized by MQTT in Home Assistant
The modelnumber on the lamp is HG07834C
In zigbee it is recognized as TS0505B
The manufacturer is not recognized  " _TZ3000_qd7hej8u"
Another guy bought the lamp also said it was also recognized as TS0505B and manufacturer LIDL and working in Home Assistant........
What can i do to get it working?
I appreciate your help.

**Comment by Like-electronics** [comment]
it is solved by loading the newest version of M2Q

**Reply by vandenbogerd** [reply]
M2Q?

**Comment by musicscoree96** [comment]
I want to use ZHA. Is there a PY file for this bulb. I bought 3 of them.

Tryed different TS0505B (and TS0505A) quircks without any luck.
